terraform/deployments/elasticsearch/README.md

```diff
@@ -0,0 +1,82 @@
+# Elasticsearch cluster
```

Comment:
On the one hand I like the idea of documenting this stuff, on the other hand I know we won't update it (without an automatic process). I'm not sure whether to say we just shouldn't do it or not, so I'll leave that decision to you 😊

Reply:
Personally I think we should automate terraform-docs usage (which is what generated this readme, I didn't actually write a single character of it :)), I've used it enough to recognise that the old module (in govuk-aws) also used it and had a generated readme).

I take this comment to mean we aren't doing it elsewhere.